site/content/docs/5.2/components/breadcrumb.md

```diff
@@ -90,6 +90,20 @@ You can also remove the divider setting `--bs-breadcrumb-divider: '';` (empty st
 $breadcrumb-divider: none;
 ```
 
+## Dark variant
+
+Breadcrumbs come with a dark variant: `.breadcrumb-dark`.
+
+{{< example class="bg-dark" >}}
```

Comment:
If you change it to `bg-warning` for example, the dark variant doesn't have the correct colors:
![Screenshot from 2022-08-24 13-15-46](https://user-images.githubusercontent.com/17381666/186405193-08b58222-5a55-499a-9123-233a3c76c0d4.png)

Reply:
Ah yes nice catch, baited by the utility once again >< Correcting it!